README.md

```diff
@@ -6,7 +6,10 @@ This is a Qt/C++ implementation of the famous game "Achtung die Kurve", accelera
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/magnus-gross/quickcurver.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/magnus-gross/quickcurver/alerts/)
 [![Language grade: C/C++](https://img.shields.io/lgtm/grade/cpp/g/magnus-gross/quickcurver.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/magnus-gross/quickcurver/context:cpp)
 
-![screenshot](http://i.imgur.com/IGa3NwA.png)
+![screenshot](https://raw.githubusercontent.com/sam9032/quickcurver/master/Screenshots/2020-04-18%2014_16_25-Quick%20Curver.png)
```

Comment:
Please don't link to your fork repository source tree. The proper way would be to upload to an external service like Imgur or into the Github upload webserver.

Reply:
> 
> 
> Also I think, one screenshot should be enough, users don't want to use up literally their entire mobile data volume if they open this Readme.

And because of that fact, i create a folder with pics, so everyone can deside to download more than one picture!